python/physicsobjects/DiLepton.py

```diff
@@ -90,7 +88,7 @@ def isSS(self):
         return int(self.lep1().charge()==self.lep2().charge())
 
     def _disp3DFromPV(self):
-        return ROOT.VertexDistance3D().distance(self.vtx(), self._pv)
+        return ROOT.VertexDistance4D().distance(self.vtx(), self._pv)
```

Comment:
why 4D?

Reply:
This is a very good question... Vinz and I both don't really remember implementing this and the blame function doesn't help us here. Do you think It would make a difference?... After realizing that CMSSW doesn't even offer VertexDistance4D... I'll change it back to 3D